src/Config_Command.php

```diff
@@ -384,5 +454,34 @@ private static function unique_key() {
 
 		return $key;
 	}
+
+	/**
+	 * Filters the values based on a provider filter key.
+	 *
+	 * @param array $values
+	 * @param array $filters
+	 * @param bool $strict
+	 *
+	 * @return array
+	 */
+	private function filter_values( $values, $filters, $strict ) {
+		$result = array();
+
+		foreach ( $values as $value ) {
+			foreach ( $filters as $filter ) {
+				if ( $strict && $filter !== $value['key'] ) {
+					continue;
+				}
+
+				if ( false === strpos( $value['key'], $filter ) ) {
```

Comment:
I see you have tests for case but was wondering should this case-insensitive as not strict? (Or maybe add a `case-insensitive` flag).

Reply:
Case-insensitivity will probably open up a huge can of worms. I'd prefer to keep everything case-sensitive for now.